booking-page/index.html

```diff
@@ -0,0 +1,146 @@
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Fredoka&display=swap"
+      rel="stylesheet"
+    />
+
+    <!-- Google Icons -->
+    <link
+      rel="stylesheet"
+      href="https://fonts.googleapis.com/css2?family=Material+Symbols+Outlined:opsz,wght,FILL,GRAD@24,400,0,0"
+    />
+  </head>
+  <body>
+    <header>
+      <div class="header">
+        <h1>Paw Patrol: The Mighty Movie - English</h1>
+        <div class="tags">
+          <p class="tag">Adventure</p>
+          <p class="tag">Animation</p>
+          <p class="tag">Comedy</p>
+        </div>
+      </div>
+    </header>
+    <main>
+      <div class="dates">
+        <div class="date active-date">
+          FRI <br />
+          <span class="day"> 13 </span><br />
+          OCT
+        </div>
+        <div class="date">
+          FRI <br />
+          <span class="day"> 13 </span><br />
+          OCT
+        </div>
+        <div class="date">
+          FRI <br />
+          <span class="day"> 13 </span><br />
+          OCT
+        </div>
+
+        <div class="date">
+          FRI <br />
+          <span class="day"> 13 </span><br />
+          OCT
+        </div>
+        <div class="date">
+          FRI <br />
+          <span class="day"> 13 </span><br />
+          OCT
+        </div>
+        <div class="date">
+          FRI <br />
+          <span class="day"> 13 </span><br />
+          OCT
+        </div>
+        <div class="date">
+          FRI <br />
+          <span class="day"> 13 </span><br />
+          OCT
+        </div>
+      </div>
+      <hr />
+      <div class="timings-section">
+        <div class="imox">
+          <span class="material-symbols-outlined"> favorite </span>
+          <h3>IMOX, Noida</h3>
+        </div>
+        <div class="info">
+          <span class="material-symbols-outlined"> info </span>
+          <p>INFO</p>
+        </div>
+        <div class="timings">
+          <div class="timing">
+            <p>8:15</p>
+          </div>
+          <div class="timing">
+            <p>11:30</p>
+          </div>
+          <div class="timing">
+            <p>14:40</p>
+          </div>
+          <div class="timing">
+            <p>15:30</p>
+          </div>
+          <div class="timing">
+            <p>20:00</p>
+          </div>
+          <div class="timing">
+            <p>23:10</p>
+          </div>
+        </div>
+      </div>
+    </main>
+    <div class="seat-btn-section hidden">
+      <button class="seat-btn">Select Seats</button>
+    </div>
+
+    <div class="modal hidden">
+      <h3 class="modal-heading">How many Seats?</h3>
+      <!-- <img src="../img/hp.jpg" width="100" alt="image" /> -->
+      <div class="seat-nums">
+        <div class="seat">1</div>
+        <div class="seat active-seat">2</div>
+        <div class="seat">3</div>
+        <div class="seat">4</div>
+        <div class="seat">5</div>
```

Comment:
These seat numbers and prices should be dynamic

Reply:
Sir, these are seat quantities actually. I am making them and prices dynamic